selenium_tests/lti_emailer/regression_suite.py

```diff
@@ -2,31 +2,36 @@
 import time
 from os import path, makedirs
 
-import HTMLTestRunner
+from selenium_common import HTMLTestRunner
 from selenium_tests.lti_emailer.emailer_permission_tests import EmailerPermissionTests
 
 
-date_timestamp = time.strftime('%Y%m%d_%H_%M_%S')
+class RegressionSuite(unittest.TestCase):
```

Comment:
Forgot to ask...why is this now a `TestCase`?  And why is all the logic now in its `main` function?